graf2d/gpadv7/inc/ROOT/TObjectDisplayItem.hxx

```diff
@@ -61,10 +63,16 @@ public:
       if (fOwner) delete fObject;
    }
 
-   void AddTColor(int color_indx, const std::string &color_value)
+   void AddColor(int color_indx, const std::string &color_value)
```

Comment:
It also replaces, right? Maybe `UpdateColor` would be a better name?

Reply:
It fully internal classes. Only reason for having it in public space - need for streamer infos.

Reply:
But even then, good names help our future selves! :-)